Python/traceback.c

```diff
@@ -512,8 +516,143 @@ _Py_DisplaySourceLine(PyObject *f, PyObject *filename, int lineno, int indent, i
     return err;
 }
 
+/* AST based Traceback Specialization
```

Comment:
I thought from the title of the PR that it's related to Mark's bytecode specializations. Are you settled on using this word? Seems a bit overloaded now.

Reply:
They are not relevant at all, but I can see the confusion. Any suggestions on alternative renaming?

Reply:
Here's how clang's documentation talks about their carets: https://clang.llvm.org/diagnostics.html

So following from there maybe something like `Precision traceback` or `Pointing traceback` because we're pointing to a specific part in the expression or trying to make it more precise?